### ISSUE TITLE
Feature Request : Don't auto create api/responses/*.js

### ISSUE THREAD

**Issue by randallmeeker** [balderdashy/sails#2959]
Instead responses should work like they work like the blueprints directory 

If ```ok.js``` found, then override the default ```ok.js``` file. 

That way the only items in /api are those we specifically create. It will make upgrading easier, because currently those files can get left behind when we upgrade our sails module. 

It also creates a consistency across the api folder.

**Comment by tjwebb** [comment]
:+1:

**Comment by sgress454** [comment]
Yeah, the decision there was based on a) wanting folks to be aware that `responses` was even a thing that you could override, and b) not forcing them to dig into the core source to find the original `ok.js` (or whatever) when they wanted to override it.  Of course, you could make the exact same argument about blueprints, except that overriding blueprints is already a lot more advanced than overriding responses, and in fact a lot of the things people *think* they need to override blueprints for, they can do by making a custom `ok.js`.

**Comment by sailsbot** [comment]
Thanks for posting, @randallmeeker.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
